packages/camera/camera_avfoundation/pubspec.yaml

```diff
@@ -20,13 +20,15 @@ dependencies:
   camera_platform_interface: ^2.7.0
   flutter:
     sdk: flutter
+  pigeon: ^18.0.0
```

Comment:
shouldn't it come as dev dependency?

Reply:
Yes, see https://github.com/flutter/packages/pull/6563